lms/static/js/edxnotes/utils/utils.js

```diff
@@ -0,0 +1,24 @@
+;(function (define, undefined) {
+'use strict';
+define([], function($, _) {
+    /**
+     * Replaces all newlines in a string by HTML line breaks.
+     * @param {String} str The input string.
+     * @return Returns `String` with '<br>' instead all newlines (\r\n, \n\r, \n and \r).
+     * @example
+     *   nl2br("This\r\nis\n\ra\nstring\r")
+     *   Output:
+     *   This<br>
+     *   is<br>
+     *   a<br>
+     *   string<br>
+     */
+    var nl2br = function (str) {
+        return (str + '').replace(/(\r\n|\n\r|\r|\n)/g, '<br>');
```

Comment:
http://en.wikipedia.org/wiki/Newline says that is used only in Mac OS up to version 9. Edx does not support it.